app/src/androidTest/java/org/simple/clinic/TestData.kt

```diff
@@ -777,4 +780,45 @@ class TestData @Inject constructor(
         type = type
     )
   }
+
+  fun encounter(
+      uuid: UUID = UUID.randomUUID(),
+      patientUuid: UUID = UUID.randomUUID(),
+      facilityUuid: UUID? = null,
+      encounteredOn: LocalDate = LocalDate.now(),
+      syncStatus: SyncStatus = SyncStatus.DONE,
+      createdAt: Instant = Instant.now(),
+      updatedAt: Instant = Instant.now(),
+      deletedAt: Instant? = null
+  ): Encounter {
+    return Encounter(
+        uuid = if (facilityUuid == null) uuid else generateEncounterUuid(facilityUuid, patientUuid, encounteredOn),
+        patientUuid = patientUuid,
+        encounteredOn = encounteredOn,
+        syncStatus = syncStatus,
+        createdAt = createdAt,
+        updatedAt = updatedAt,
+        deletedAt = deletedAt
+    )
+  }
+
+  fun encounterPayload(
+      uuid: UUID = UUID.randomUUID(),
+      patientUuid: UUID = UUID.randomUUID(),
```

Comment:
Same as above for `uuid` and `patientUuid`

Reply:
Even though I have made this change, but in the next PR I will have to revert this.
We need the uuid params to have default values because this test data is used in instrumentation tests.

Reply:
Can we not supply `UUID.random()` in the instrumentation tests alone? Otherwise, it is easy for us to still follow this pattern where failing tests make our lives miserable.

1. All UUIDs should be explicit by default.
2. UUIDs used for integration tests should explicitly use `UUID.random()`.

Does this seem reasonable?

Reply:
This makes sense. Will take care of this in the upcoming PR.